pkg/services/live/pushurl/values.go

```diff
@@ -5,9 +5,10 @@ import (
 	"strings"
 )
 
+// StableSchemaFromValues assumes stable
 func StableSchemaFromValues(values url.Values) bool {
 	key := "gf_live_stable_schema"
-	return strings.ToLower(values.Get(key)) == "true" || values.Get(key) == "1"
+	return !(strings.ToLower(values.Get(key)) == "false" || values.Get(key) == "0")
```

Comment:
@FZambia -- this changes the default to assume things are stable and requires an explicit flag to say they are not.

Reply:
Need to rename `gf_live_stable_schema` also then and fix caller's logic accordingly